spec/syntax.md

```diff
@@ -452,6 +452,9 @@ Otherwise, the set of characters allowed in names is large.
 The _nmtoken_ token doesn't have _name_'s restriction on the first character
 and is used as variant keys and option values.
 
+The _argtoken_ is used for expression arguments,
+and matches _nmtoken_ except for not allowing `-` as a first character.
+
 _Note:_ The Name and Nmtoken symbols are intentionally defined to be
 the same as XML's [Name](https://www.w3.org/TR/xml/#NT-Name) and [Nmtoken](https://www.w3.org/TR/xml/#NT-Nmtokens)
 in order to increase the interoperability with data defined in XML.
```

Comment:
code quote `Name` and `Nmtoken` and match the casing in the ABNF. I wouldn't call them "symbols". I would probably call them productions (since that's the ABNF terminology) or just "nothing", i.e. "_Note:_ `name` and `nmtoken` are intentionally defined..."

I would also be more direct here, changing "... in order to increase the interoperability with data defined in XML" to something more like:

> ... in order to ensure current and future compatibility with LDML (UAX35) and it's various names and values, since these are defined using XML.

Reply:
Should these changes be done in a separate non-normative editorial PR?